src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = days * 40;
+
+  if (days >= 3 && days < 7) {
+    cost -= 20;
+  }
+
+  if (days >= 7) {
+    cost -= 50;
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item, create variables for main values - 20, 50, 40